brainframe_qt/ui/main_window/activities/stream_activity/stream_view_ui.py

```diff
@@ -57,12 +55,8 @@ def _init_layout(self) -> None:
 
         # https://stackoverflow.com/a/43835396/8134178
         # 1 : 4 width ratio when expanded
-        # I do this in a single-shot so that .width() can calculate after sizes
-        # have been initialized
-        set_splitter_sizes = functools.partial(
-            self.splitter.setSizes,
-            [.20 * self.width(), .80 * self.width()])
-        QTimer.singleShot(0, set_splitter_sizes)
+        x = 1000000
```

Comment:
Heh so what goes on here, compared to the previous code?

Reply:
See #2 here in this response: https://stackoverflow.com/a/43835396/8134178

My goal is to make the QSplitter split based on percent width (until the user adjusts it), which I guess is not officially supported. The workaround is to use QSplitter.setSizes, but it takes real pixels (sizing down to fit available area).

Previously I was waiting one event loop cycle, under the assumption the widget would be populated, before calculating the size of the QSplitter sections. This turned out to be naive (1 cycle is not longer long enough) and unnecessary (I can just use a really large number and it will get scaled down proportionally).

I forgot to give the `x` a better name, so I'll do that now

Reply:
Got it, thanks for the explanation!